Examples/SnapinsSDKExample/app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="colorPrimary">#3F51B5</color>
```

Comment:
Great. Thanks. I did it the first way, but add a <!-- Comment --> in the XML about where to find branding instructions for the SDK...